roles/openshift_storage_ceph/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+openshift_storage_ceph_timeout: 600
+openshift_storage_ceph_mon_count: 1
+openshift_storage_ceph_use_host_network: False
+openshift_storage_ceph_dirs: "/var/lib/rook"
+openshift_storage_ceph_dashboard: True
+openshift_storage_ceph_dashboard_url_prefix: "/ceph-dashboard"
+openshift_storage_ceph_image: "{{ openshift_storage_ceph_image_dict[openshift_deployment_type] | lib_utils_oo_oreg_image((oreg_url | default('None'))) }}"
```

Comment:
This is where `registry.svc.ci.openshift.org` gets injected - CI has this domain set as oreg_url. 

`registry.redhat.io` should be uncommented for OSE and `openshift_storage_ceph_image` should not use `lib_utils_oo_oreg_image` filter for Origin

Reply:
I'm not sure to understand how to overcome this, all the other existing roles are using this pattern. Does that mean they have images in `registry.svc.ci.openshift.org`?

How should I uncomment `registry.redhat.io`, thanks for your help.

Reply:
`oreg_url` sets a dedicated registry which should have the copy of all images for dedicated installs. Origin may use some images from docker.io, so in some cases its fine not to replace the registry (this is done for grafana, for instance). This behaviour can be overriden via custom `openshift_storage_ceph_image`. 

This does not apply in OSE case though, as it has a strict policy on upstream images.

I think
```
openshift_storage_ceph_image: "{{ openshift_storage_ceph_image_dict[openshift_deployment_type] }}"
openshift_storage_ceph_image_dict:
  origin: "docker.io/ceph/ceph:v13"
  openshift-enterprise: "{{ 'registry.redhat.io/rhceph/rhceph-3-rhel7:latest' | lib_utils_oo_oreg_image((oreg_url | default('None'))) }}"
```
would work here

Reply:
Thanks a lot! Testing this now :)